frontend/src/components/activePage/TableInstance/RowInstanceActions/RowInstanceActionsExtended.tsx

```diff
@@ -90,28 +94,27 @@ const RowInstanceActionsExtended: FC<IRowInstanceActionsExtendedProps> = ({
             </Button>
           </span>
         </Tooltip>
-
-        {fileManager && (
-          <Tooltip placement="top" title={'File Manager'}>
-            <Upload name="file">
-              <span
-                className={`${
-                  status !== 'VmiReady' ? 'cursor-not-allowed' : ''
-                }`}
-              >
-                <Button
-                  shape="circle"
-                  className={`hidden mr-3 xl:inline-block ${
-                    status !== 'VmiReady' ? 'pointer-events-none' : ''
-                  }`}
-                  disabled={status !== 'VmiReady'}
-                >
-                  <FolderOpenOutlined />
-                </Button>
-              </span>
-            </Upload>
-          </Tooltip>
-        )}
+        <Tooltip
+          title={getFileManagerTooltipText(
+            status === 'VmiReady',
+            environmentType!
+          )}
+        >
+          <span
+            className={`${fileManagerDisabled ? 'cursor-not-allowed' : ''}`}
+          >
+            <Button
+              shape="circle"
+              className={`hidden mr-3 xl:inline-block ${
+                fileManagerDisabled ? 'pointer-events-none' : ''
+              }`}
+              disabled={fileManagerDisabled}
+              onClick={() => window.open(`${url}/mydrive/files`, '_blank')}
```

Comment:
```suggestion
              href="`${url}/mydrive/files`"
              target="_blank"
```
It's better to use the link mode of the button. This also prevents issues with popup blockers and similar stuff.
I'd ask you to replace this behavior in other places if it's used anywhere else.

Reply:
Also, do we want to try opening nextcloud with vms? (see [the slack](https://crown-team-group.slack.com/archives/C01J519B83G/p1636997434075600))

Reply:
i fix it everywhere

Reply:
ok for the suggestion about nextcloud